Runtime/JSValue.cs

```diff
@@ -156,17 +156,21 @@ public static JSValue CreateFunction(string name, JSCallback callback)
         return CreateFunction(utf8Name, callback);
     }
 
-    public static JSValue CreateError(JSValue code, JSValue message)
-        => napi_create_error(Env, (napi_value)code, (napi_value)message, out napi_value result).ThrowIfFailed(result);
+    public static JSValue CreateError(JSValue? code, JSValue message)
+        => napi_create_error(Env, code.AsNapiValueOrNull(), (napi_value)message,
+            out napi_value result).ThrowIfFailed(result);
 
-    public static JSValue CreateTypeError(JSValue code, JSValue message)
-        => napi_create_type_error(Env, (napi_value)code, (napi_value)message, out napi_value result).ThrowIfFailed(result);
+    public static JSValue CreateTypeError(JSValue? code, JSValue message)
+        => napi_create_type_error(Env, code.AsNapiValueOrNull(), (napi_value)message,
+            out napi_value result).ThrowIfFailed(result);
 
-    public static JSValue CreateRangeError(JSValue code, JSValue message)
-        => napi_create_range_error(Env, (napi_value)code, (napi_value)message, out napi_value result).ThrowIfFailed(result);
+    public static JSValue CreateRangeError(JSValue? code, JSValue message)
+        => napi_create_range_error(Env, code.AsNapiValueOrNull(), (napi_value)message,
+            out napi_value result).ThrowIfFailed(result);
 
-    public static JSValue CreateSyntaxError(JSValue code, JSValue message)
-        => node_api_create_syntax_error(Env, (napi_value)code, (napi_value)message, out napi_value result).ThrowIfFailed(result);
+    public static JSValue CreateSyntaxError(JSValue? code, JSValue message)
+        => node_api_create_syntax_error(Env, code.AsNapiValueOrNull(), (napi_value)message,
```

Comment:
Do these APIs not accept `undefined` as valid values for the code?

In JS, `new Error().code === undefined`.

Reply:
In Node.JS they expect either `null` or a `string`: https://github.com/nodejs/node/blob/655b070d31efeea2f5580ca221167f8abc43f0e8/src/js_native_api_v8.cc#L1583